epersongroups.md

```diff
@@ -0,0 +1,307 @@
+
+Retrieves direct child epeople of the given group.
+Requesting indirect child epeople is not supported
+
+Sample:
+* GET /rest/api/eperson/groups/3b1de75d-5bf9-4ca3-bf4d-42bc8abd0d42/epersons
+
+
+```json
+{
+    "_embedded": {
+      "epersons": [
+        {
+          "id": "a6086b34-3918-45b7-8ddd-9329a702a26a",
+          "uuid": "a6086b34-3918-45b7-8ddd-9329a702a26a",
+          "name": "atmirenv@gmail.com",
+          "handle": null,
+          "metadata": {
+            "eperson.firstname": [
+              {
+                "value": "John",
+                "language": null,
+                "authority": "",
+                "confidence": -1,
+                "place": 0
+              }
+            ],
+            "eperson.language": [
+              {
+                "value": "en",
+                "language": null,
+                "authority": "",
+                "confidence": -1,
+                "place": 0
+              }
+            ],
+            "eperson.lastname": [
+              {
+                "value": "Doe",
+                "language": null,
+                "authority": "",
+                "confidence": -1,
+                "place": 0
+              }
+            ]
+          },
+          "netid": null,
+          "lastActive": "2019-09-25T15:59:28.000+0000",
+          "canLogIn": true,
+          "email": "user@institution.edu",
+          "requireCertificate": false,
+          "selfRegistered": false,
+          "groups": null,
+          "type": "eperson",
+          "_links": {
+            "self": {
+              "href": "https://dspace7.4science.it/dspace-spring-rest/api/eperson/epersons/a6086b34-3918-45b7-8ddd-9329a702a26a"
+            }
+          }
+        }
+      ]
+    },
+    "_links": {
+      "self": {
+        "href": "https://dspace7.4science.it/dspace-spring-rest/api/eperson/groups/3b1de75d-5bf9-4ca3-bf4d-42bc8abd0d42/epersons"
+      }
+    },
+    "page": {
+      "number": 0,
+      "size": 10,
+      "totalPages": 1,
+      "totalElements": 10
+    }
+}
+```
+
+### Add an EPerson to a parent Group
+
+**POST /api/eperson/groups/<:groupuuid>/epersons**
+
+To add an eperson to a parent group, perform a POST to the Epeople of a Group endpoint when logged in as admin.
+
+The actual eperson is part of the body using the uri-list
+
+Example:
+```bash
+curl -i -X POST "https://dspace7.4science.it/dspace-spring-rest/api/eperson/groups/617cf46b-535c-42d5-9d22-327ce2eff6dc/epersons"
+  -H "Content-Type:text/uri-list"
+  -d "https://dspace7.4science.it/dspace-spring-rest/api/eperson/epersons/a6086b34-3918-45b7-8ddd-9329a702a26a"
+```
+
+The eperson is mandatory
+
+Return codes:
+* 204: if the update succeeded 
+* 401 Forbidden - if you are not authenticated
+* 403 Unauthorized - if you are not logged in with sufficient permissions 
+* 422: if the specified eperson is not found
+
+### Remove an EPerson from a parent Group
+
+**PUT /api/eperson/groups/<:groupuuid>/epersons**
```

Comment:
Again, I think this PUT should be in the section above next to the POST

Reply:
I've moved this to make it more clear which methods are supported and which not for changing the epeople